tests/commands/test_groups.py

```diff
@@ -135,6 +141,18 @@ def test_groups_group_command__create_HTTPError(mock_client, capfd):
     assert "Matching groups (3)" in capture.out
 
 
+def test_groups_group_command__funding_sources(mock_client, capfd):
+    args = Namespace(group_command="funding_sources")
+    mock_client.get_concession_group_linked_funding_sources.return_value = (r for r in GROUP_FUND_RESPONSES)
+    res = groups(args)
+
+    capture = capfd.readouterr()
+    assert res == RESULT_SUCCESS
+    assert "Listing funding sources for:" in capture.out
+    assert "Funding Source:" in capture.out
+    assert "Matching groups (3)" in capture.out
+
+
```

Comment:
This test looks good for covering the command's happy path

I suggest adding one more test named `test_groups_group_command__funding_sources_HTTPError` to cover the case where an `HTTPError` is thrown. You can refer to any of the tests in this file that end in `_HTTPError` as an example.

The coverage report shows there is no test coverage for these lines: [189-191 in `groups.py`](https://github.com/cal-itp/littlepay/pull/55/files#diff-24575a0adbb3b1bbf7423a2a1e2b164a4fac5a02175f717feed94fd7540a73c6R189-R191)

![image](https://github.com/cal-itp/littlepay/assets/25497886/15ae93a9-0184-44ba-8666-a001abb86f98)

Reply:
Ah I forgot about `test_groups_group_command__funding_sources_HTTPError`, thanks for the reminder! I'll add this test.